lms/djangoapps/grades/new/course_grade_factory.py

```diff
@@ -66,7 +66,15 @@ def read(self, user, course=None, collected_block_structure=None, course_structu
             else:
                 return None
 
-    def update(self, user, course=None, collected_block_structure=None, course_structure=None, course_key=None):
+    def update(
+            self,
+            user,
+            course=None,
+            collected_block_structure=None,
+            course_structure=None,
+            course_key=None,
+            force_update_subsections=False,
```

Comment:
We need to pass it in because we also call `update` when a subsection grade is updated. If we assume `true` for all cases, then all subsection grades will update whenever a learner submits a problem.

https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/signals/handlers.py#L246